### PR TITLE
Uptime tracking api

### PR DESCRIPTION
**Pull request by ceyonur**
## Why this should be merged

Adds Validators API (GetCurrentValidators) 

## How this works

Refactors API (RPC) and Service files, adds new handler for `/validators` endpoint and implement `GetCurrentValidators` API

## How this was tested

Locally

## How is this documented

Need to be documented

### CODE REVIEW

File: plugin/evm/api.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
```
Comment (darioush) [inline plugin/evm/api.go:2]: should we leave this in `service.go` and add the new stuff in `service_validator.go`?

or we can rename this to something like `api_snowman.go` and have `api_validator.go`?

File: plugin/evm/vm.go [hunk 2]
```diff
@@ -1082,6 +1083,16 @@ func (vm *VM) CreateHandlers(context.Context) (map[string]http.Handler, error) {
...
+	if vm.config.ValidatorsAPIEnabled {
+		validatorsAPI, err := newHandler("validators", &ValidatorsAPI{vm})
+		if err != nil {
+			return nil, fmt.Errorf("failed to register service for admin API due to %w", err)
+		}
+		apis[validatorsEndpoint] = validatorsAPI
+		enabledAPIs = append(enabledAPIs, "validators")
+	}
+
+	// RPC APIs
```
Comment (darioush) [inline plugin/evm/vm.go:1095]: nit: not sure which RPC this comment is referring to?

File: plugin/evm/vm.go [hunk 3]
```diff
@@ -1082,6 +1083,16 @@ func (vm *VM) CreateHandlers(context.Context) (map[string]http.Handler, error) {
 		enabledAPIs = append(enabledAPIs, "subnet-evm-admin")
 	}
 
+	if vm.config.ValidatorsAPIEnabled {
+		validatorsAPI, err := newHandler("validators", &ValidatorsAPI{vm})
+		if err != nil {
+			return nil, fmt.Errorf("failed to register service for admin API due to %w", err)
```
Comment (darioush) [inline plugin/evm/vm.go:1089]:
```suggestion
			return nil, fmt.Errorf("failed to register service for validators API due to %w", err)
```

**Review by darioush — DISMISSED** [review]

File: plugin/evm/vm.go (hunk 2 above)
Comment (ceyonur) [reply plugin/evm/vm.go:1095]: `/rpc` endpoint

File: plugin/evm/api.go (hunk 1 above)
Comment (ceyonur) [reply plugin/evm/api.go:2]: My intention was to keep avalancheRPC server handlers separate from `/rpc` handler, and follow avalanche endpoints -> service, /rpc endpoint -> api convention.

**Review by michaelkaplan13 — APPROVED** [review]
LGTM, only question is if there is an existing E2E test suite that we could add a new test case to in order to confirm the new `/validators` endpoints is working as expected in CI?

**Review by michaelkaplan13 — APPROVED** [review]

**Pull request merged**